src/agent/src/mount.rs

```diff
@@ -872,17 +993,21 @@ pub async fn add_storages(
         let logger = logger.new(o!(
             "subsystem" => "storage",
             "storage-type" => handler_name.to_owned()));
-
+        if !storage
+            .options()
+            .contains(&"io.katacontainers.image.tarfs".to_string())
```

Comment:
We should restrict the usage of option-specific code to as few places as possible.

I don't think using it here is warranted.